src/nvim/screen.c

```diff
@@ -2216,9 +2234,10 @@ win_line (
 
   bool search_attr_from_match = false;  // if search_attr is from :match
   bool has_bufhl = false;               // this buffer has highlight matches
-  int bufhl_attr = 0;                   // attributes desired by bufhl
   BufhlLineInfo bufhl_info;             // bufhl data for this line
 
+  bool do_eoltext = false;
```

Comment:
does it make sense to rename this to `do_virttext`? Also could group it with the other "bufhl" variables by moving it up 1 line.